runtime-image/image.yaml

```diff
@@ -0,0 +1,74 @@
+schema_version: 1
+
+name: "jboss-eap-7-tech-preview/eap74-openjdk11-runtime-openshift-rhel8"
+description: "The JBoss EAP 7.4.0.Beta openjdk11 runtime image"
+version: "7.4.0.Beta"
+from: "registry.redhat.io/ubi8/ubi:latest"
+labels:
+    - name: "com.redhat.component"
+      value: "jboss-eap-74-beta-openjdk11-runtime-openshift-rhel8-container"
+    - name: "io.k8s.description"
+      value: "Base image to run an EAP server and application"
+    - name: "io.k8s.display-name"
+      value: "JBoss EAP runtime image"
+    - name: "io.openshift.expose-services"
+      value: "8080:http"
+    - name: "io.openshift.tags"
+      value: "javaee,eap,eap7"
+    - name: "maintainer"
+      value: "Red Hat"
+envs:
+    - name: "LAUNCH_JBOSS_IN_BACKGROUND"
+      value: "true"
+    - name: HTTPS_ENABLE_HTTP2
+      value: "true"
+    - name: SCRIPT_DEBUG
+      description: If set to true, ensures that the bash scripts are executed with the -x option, printing the commands and their arguments as they are executed.
+      example: "true"
+    - name: JBOSS_HOME
+      value: "/opt/eap"
+    # env var set by modules in the builder image context, duplicating them here.
+    - name: JBOSS_MODULES_SYSTEM_PKGS
+      value: "org.jboss.logmanager,jdk.nashorn.api"
+    - name: DEFAULT_ADMIN_USERNAME
+      value: "eapadmin"
+    - name: MICROPROFILE_CONFIG_DIR_ORDINAL
```

Comment:
The EAP 7.4.0.Beta images no longer contain microprofile-config. This env var is not needed anymore